### PR TITLE
Add ledger package.

### PR DESCRIPTION
**Pull request by lloydhuang**
Hello Admin,
I added a ledger package - A double-entry accounting system. It is great tools for emacs ledger-mode. This is my first time to package,  If any problem, pls feel free to let me know, thank you.

BRs
Lloyd

### CODE REVIEW

File: packages/ledger.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     x86_64: '',
+  })
+
+  depends_on 'gmp' => :build
+  depends_on 'boost' => :build
+  depends_on 'mpfr' => :build
+
+  def self.build
+    system "./acprep"
+    system 'make'
```
Comment (uberhacker) [inline packages/ledger.rb]: We will probably want to add `prefix=#{CREW_PREFIX}` and possibly `libdir=#{CREW_LIB_PREFIX}` to abstract the install location.

File: packages/ledger.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  binary_sha256 ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+
+  depends_on 'gmp' => :build
+  depends_on 'boost' => :build
+  depends_on 'mpfr' => :build
```
Comment (uberhacker) [inline packages/ledger.rb]: All of these dependencies come along for the ride in Chromebrew "core", so they aren't needed here.

**Comment by uberhacker** [comment]
Thank you for the new package @lloydhuang!  Please see comments in the `Files changed` section.

File: packages/ledger.rb (hunk 2 above)
Comment (uberhacker) [reply packages/ledger.rb]: Actually, `boost` is not in "core" so that line is fine.

File: packages/ledger.rb (hunk 1 above)
Comment (lloydhuang) [reply packages/ledger.rb]: Option --prefix=#{CREW_PREFIX} is OK. But this package don't provide --libdir option. Is okay that just add --prefix option?

Comment (uberhacker) [reply packages/ledger.rb]: If no `--libdir` configure option is available, that usually indicates there are no library files included in this app.  It is fine to use only `--prefix`.

**Comment by JL2210** [comment]
@ubethacker If this requires Boost as a runtime library, then that should be a regular dependency, not a build-time one.

**Comment by lloydhuang** [comment]
@uberhacker Please help to review it, Thank you.

**Pull request merged**